dash-pipeline/Makefile

```diff
@@ -672,4 +672,4 @@ run-saichallenger-tutorials: deploy-ixiac
 
 install-docker-compose:
 	sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$$(uname -s)-$$(uname -m)" -o /usr/local/bin/docker-compose
-	sudo chmod +x /usr/local/bin/docker-compose
+	sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
How about restore this file so it's not part of commit?

Reply:
I tried copying from the main branch (which is in sync with upstream) to working branch to see if it restores it but it did not. What is the best way to restore only this file back to version of the upstream branch?

Reply:
You can google for "git restore a file from master" but I think the syntax would be:
```
git checkout origin/main dash-pipeline/Makefile
```